src/storage/mvcc/txn.rs

```diff
@@ -2403,4 +2403,15 @@ mod tests {
             e => panic!("unexpected error: {}", e),
         };
     }
+
+    #[test]
+    fn test_commit_pessimistic_lock() {
+        let engine = TestEngineBuilder::new().build().unwrap();
+
+        let k = b"k";
+        must_acquire_pessimistic_lock(&engine, k, k, 10, 10);
+        must_commit_err(&engine, k, 20, 30);
```

Comment:
What does this mean?

Reply:
Shouldn't commit a pessimistic lock with different start_ts.

Reply:
It is covered by a normal commit. I'm fine with it, though.